projects/plugins/jetpack/modules/comments/subscription-modal-on-comment/subscription-modal.css

```diff
@@ -0,0 +1,153 @@
+.jetpack-subscription-modal {
+	visibility: hidden;
+	position: fixed;
+    z-index: 50000;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+	opacity: 0;
+    transition: visibility 0s, opacity 0.5s ease;
+    display: flex;
+    justify-content: space-evenly;
+    align-content: center;
+    flex-wrap: wrap;
+}
+
+.jetpack-subscription-modal.open {
+	opacity: 1;
+	background-color: rgba(0,0,0,0.3);
+	visibility: visible;
+}
+
+.jetpack-subscription-modal__modal-content {
+	text-align: center;
+	background-color: #fefefe;
+	width: 100%;
+	max-width: 650px;
+	box-sizing: border-box;
+	transition: visibility 0s, opacity 0.3s linear;
+	opacity: 0;
+	display: flex;
+	flex-direction: column;
+	gap: 5px;
+	justify-content: center;
+}
+
+.jetpack-subscription-modal.open .jetpack-subscription-modal__modal-content {
+	opacity: 1;
+	top: 0;
+	visibility: visible;
+}
+
+/* Hide the modal content when iframe is present */
+.jetpack-subscription-modal.has-iframe .jetpack-subscription-modal__modal-content {
+	background: transparent;
+}
+
+.jetpack-subscription-modal.has-iframe .jetpack-subscription-modal__modal-content-form {
+	visibility: hidden;
+	opacity: 0;
+}
+
+.jetpack-subscription-modal__modal-content .jetpack-subscription-modal__iframe-container {
+	visibility: hidden;
+	opacity: 0;
+	background-image: url( 'https://s0.wp.com/i/loading/dark-200.gif' );
+	background-size: 50px;
+	background-repeat: no-repeat;
+	background-position: center 150px;
+	margin: 0 !important;
+	box-shadow: none;
+	-webkit-box-shadow: none;
+	-moz-box-shadow: none;
+	border: none;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	top: 0;
+	width: 100% !important;
+	height: 100%;
+	position: absolute;
+}
+
+.jetpack-subscription-modal.has-iframe .jetpack-subscription-modal__iframe-container {
+	opacity: 1;
+	visibility: visible;
+
+}
+
+.jetpack-subscription-modal__iframe {
+	margin: 0 !important;
+	height: 100% !important;
+	width: 100% !important;
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+}
+
+/*
+ * These text-wrap properties still have limited browser
+ * support, but based on feedback still adding them for when
+ * they are supported.
+ */
+.jetpack-subscription-modal__modal-content p {
+	text-wrap: balance;
```

Comment:
I think you could apply balance for the modal's heading, and leave `pretty` for the `p`. I think it's fine to not stack them together as browser support is pretty good at this point, and only Firefox is lacking (but presumably not for long).

More in p58i-g2L-p2